src/middleware/authentication_middleware_sole_trader.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, Response } from "express";
+import { AuthOptions, acspProfileCreateAuthMiddleware } from "@companieshouse/web-security-node";
+
+import { CHS_URL } from "../utils/properties";
+import { BASE_URL, CHECK_SAVED_APPLICATION } from "../types/pageURL";
+
+export const authenticationMiddlewareForSoleTrader = (req: Request, res: Response, next: NextFunction) => {
+
+    const authMiddlewareConfig: AuthOptions = {
+        chsWebUrl: CHS_URL,
+        returnUrl: BASE_URL + CHECK_SAVED_APPLICATION
```

Comment:
Return url would be different than check saved application if only sole trader verification is enabled.